NewEvaluation.css

```diff
@@ -54,8 +57,29 @@ body{
 }
 
 
+.input-candidate{
+	width:33%;
+	border: 0;
+	padding: 0;
+	vertical-align: middle;
+	height: 2em;
+	/*box-shadow: 0px 1px 0px #888888; */
```

Comment:
I have to 100% agree here but there is a twist. I think this is the part which is dedicated to the newEvaluation.html part where I made a linear input form. The issue here is that I think I had a problem with underlining the text properly and the code was running fine on Chrome but was not doing anything on Mozilla. This issue occurred during the end of the training session and we had not that much time for extra stuff to do so I let it like that with the intention of asking the next day(as u might have seen my project is just 2 out of 3 pages polished).